src/helpers/order-process/order-process-helper.ts

```diff
@@ -18,6 +19,21 @@ export const listOrderProcesses = (
     `${CATALOG_API_BASE}/order_processes?filter[name][contains_i]=${filter}&limit=${limit}&offset=${offset}`
   );
 
+export const loadProductOptions = (
+  filterValue = ''
+): Promise<ApiCollectionResponse<PortfolioItem>> => {
```

Comment:
This does not return `ApiCollectionResponse<PortfolioItem>`.

We can create a new common type form select options. Should look something like this:
```TS
export interface SelectOption {
  label: string;
  value?" any;
}

export type SelectOptions = SelectOption[]
```